scripts/verify-storage-persistence.sh

```
@@ -0,0 +1,874 @@
+    echo -e "${GREEN}✓ Authenticated${NC}"
+    
+    echo ""
+    echo -e "${MAGENTA}2. Creating and verifying API key...${NC}"
+    
+    local key_info
+    key_info=$(create_api_key "$OC_TOKEN" | tail -1)
+    
+    local api_token api_key_jti api_key_name
+    api_token=$(echo "$key_info" | jq -r '.token')
+    api_key_jti=$(echo "$key_info" | jq -r '.jti')
+    api_key_name=$(echo "$key_info" | jq -r '.name')
```

Shellcheck (0.11.0)

[warning] api_key_name appears unused. Verify use (or export if used externally).

(SC2034)